request/node.js

```diff
@@ -92,6 +92,12 @@ define([
 					clearTimeout(timeout);
 				}
 				response.text = body.join('');
+				if (response.status >= 400){
```

Comment:
This should use `dojo/request/util.checkStatus(response.status)` instead of the check used here.